src/RwaFlipper.sol

```diff
@@ -16,22 +19,36 @@ contract RwaFlipper {
     }
 
     VatLike public vat;
+    CatLike public cat;
     bytes32 public ilk;
```

Comment:
Also should be immutable to save gas.

Reply:
Can't make this one immutable for reasons described above, but it's also File-able, so needs to remain editable.

Reply:
The `cat` can be filed, but not the `ilk`.

Reply:
`immutable` isn't available on this version of the compiler yet.